README.md

```diff
@@ -24,8 +24,7 @@ We've set this up in [Rubular for you here](http://rubular.com/r/s2jizA9W9V). Yo
 
 ## Writing Regular Expressions
 
-In Ruby, regular expressions are always written between forward slashes:
-`/your regex/`. In Rubular, you can see that these slashes have already been written for you.
+In Ruby, regular expressions are generally written between forward slashes: `/your regex/`. This is the 'literal' alternative to creating a regular expression object using the following syntax: `Regexp::new('your regex')`. In Rubular, you can see that these slashes have already been written for you.
```

Comment:
Ok will change. Otherwise good to merge, or should this explanation be lengthier?